projects/bitcoin-core/build.sh

```diff
@@ -44,8 +44,8 @@ fi
 if [ "$FUZZING_ENGINE" = "libfuzzer" ]; then
   CONFIG_SITE="$PWD/depends/$BUILD_TRIPLET/share/config.site" ./configure --enable-fuzz --with-sanitizers=fuzzer
```

Comment:
I was hoping to get rid of this special case and replace this with 
```suggestion
  CONFIG_SITE="$PWD/depends/$BUILD_TRIPLET/share/config.site" ./configure --enable-fuzz SANITIZER_LDFLAGS="$LIB_FUZZING_ENGINE"
```

as well